docs/pages/database-access/guides/redis-aws.mdx

```diff
@@ -0,0 +1,235 @@
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Create the Database Service configuration:
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=teleport.example.com:3080 \
+   --token=/tmp/token \
+   --elasticache-discovery=us-west-1
+```
+
+</ScopedBlock>
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+$ teleport db configure create \
+   -o file \
+   --proxy=mytenant.teleport.sh \
+   --token=/tmp/token \
+   --elasticache-discovery=us-west-1
+```
+
+</ScopedBlock>
+
+The command will generate a Database Service configuration with ElastiCache
+database auto-discovery enabled on the `us-west-1` region and place it at the
+`/etc/teleport.yaml` location. See [Configuration
+Reference](../reference/configuration.mdx) for more details if you wish to
+customize the configuration.
+
+## Step 4/7. Create an IAM policy for Teleport
+
+Teleport needs AWS IAM permissions to be able to:
+
+- Discover and register ElastiCache Redis clusters.
+- Modify ElastiCache user passwords for Teleport managed users and
+  save the passwords in AWS Secrets Manager. 
+
+(!docs/pages/includes/database-access/aws-bootstrap.mdx!)
+
+## Step 5/7. Start the Database Service
+
+Start the Database Service:
+
+```code
+$ teleport start --config=/etc/teleport.yaml
+```
+
+The Database Service will discover and register all ElastiCache Redis clusters
+according to the configuration.
+
+<Admonition type="note" title="AWS credentials">
+  The Teleport Database Service uses the default credential provider chain to
+  find AWS credentials. See [Specifying
+  Credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+  for more information.
+</Admonition>
+
+## Step 6/7. Create a Teleport managed ElastiCache user (optional)
+
+To provide better security, it is recommended to use [Redis
+ACL](https://redis.io/docs/manual/security/acl/) for authentication with Redis
+and let Teleport manage the users. The Database Service constantly rotates
+Teleport managed user's password, saves the password in AWS Secrets Manager,
+and automatically sends `AUTH` with the saved password when connecting the
+client to the Redis server.
+
+To enable Redis ACL, please see [Authenticating users with Role-Based Access
+Control for
+ElastiCache](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Clusters.RBAC.html).
+
+Once an ElastiCache user is created with the desired access, add an
+AWS resource tag `teleport.dev/managed` with the value `true` to this user. The
+Database Service will automatically discover this user if it is associated with
+a registered database. Keep in mind that it may take the Database Service some
+time (up to 20 minutes) to discover this user once the tag is added.
+
+<Admonition type="tip">
+It is not required for all users in the same ElastiCache user group to be
+managed by Teleport. Just the tag users that are desired to be used for
```

Comment:
I think we can start by asking when a reader would be tempted to assume that all users in the same ElastiCache user group must be managed by Teleport. Are there any details in this section that would lead a reader to believe that? One alternative could be to make those details clearer.

Reply:
Agreed that most likely readers wouldn't suspect all ElastiCache has to be managed. I've removed this tip entirely.